libraries-2/pom.xml

```diff
@@ -119,22 +117,40 @@
             <artifactId>spring-jdbc</artifactId>
             <version>${spring.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.sleepycat</groupId>
+            <artifactId>je</artifactId>
+            <version>18.3.12</version>
+        </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M7</version>
```

Comment:
can we use the property from the main pom to define the version here, or is the 3.0.0-M7 required?

Reply:
Done I added the variable of **maven-surefire-plugin.version**.